src/main/java/seedu/address/logic/commands/EditBuyerCommand.java

```diff
@@ -108,8 +109,11 @@ private static Buyer createEditedPerson(Buyer buyerToEdit, EditBuyerDescriptor e
                 .orElse(buyerToEdit.getDesiredCharacteristics().orElse(null));
         Priority updatedPriority = editBuyerDescriptor.getPriority().orElse(buyerToEdit.getPriority());
 
+        // Should time be updated
+        LocalDateTime entryTime = buyerToEdit.getEntryTime();
```

Comment:
I think maybe we don't need this because we aren't allowing the user to edit the entry time

Reply:
Ok!